tools/csv-merger/csv-merger.go

```diff
@@ -623,6 +623,9 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	// This can be used to enable/disable console support for any supported feature
 	// Example: "features.ocs.openshift.io/enabled": `["external", "foo1", "foo2", ...]`
 	ocsCSV.Annotations["features.ocs.openshift.io/enabled"] = `["external"]`
+	// Used by UI to validate user uploaded metdata
+	// Metadata is used to connect to an external cluster
+	ocsCSV.Annotations["external.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["rook-ceph-retain-bucket"]}`
```

Comment:
This is still a bad URI, as it does not follow the pattern established by the line above. Please change to `external.features.ocs.openshift.io/validation`.